test/runtime_tests.cc

```diff
@@ -1898,5 +1898,62 @@ namespace detail {
 		CHECK(out == 43);
 	}
 
+	TEST_CASE("accessors deduces correctly from SYCL 2020 tags for device", "[accessor]") {
```

Comment:
This test handles more than just tag types. Consider renaming to something like "accessor correctly deduces access mode from SYCL 2020 tag types and no_init property".

Reply:
done